R/ReactantExtra/build_tarballs.jl

```diff
@@ -0,0 +1,222 @@
+	
+#	mkdir -p "$JULIA_PATH"; 
+#	tar -xzf julia.tar.gz -C "$JULIA_PATH" --strip-components 1; 
+#	rm julia.tar.gz; 
+
+# cd ..
+
+mkdir .bazhome
+export HOME=`pwd`/.bazhome
+
+export BAZEL_CXXOPTS="-std=c++17"
+BAZEL_FLAGS=()
+BAZEL_BUILD_FLAGS=()
+
+# don't run out of temporary space
+BAZEL_FLAGS+=(--output_user_root=/workspace/bazel_root)
+
+BAZEL_BUILD_FLAGS+=(--jobs ${nproc})
+
+BAZEL_BUILD_FLAGS+=(--verbose_failures)
+BAZEL_BUILD_FLAGS+=(--cxxopt=-std=c++17 --host_cxxopt=-std=c++17)
+BAZEL_BUILD_FLAGS+=(--cxxopt=-DTCP_USER_TIMEOUT=0)
+BAZEL_BUILD_FLAGS+=(--check_visibility=false)
+BAZEL_BUILD_FLAGS+=(--build_tag_filters=-jlrule)
+BAZEL_BUILD_FLAGS+=(--experimental_cc_shared_library)
+
+# Disable enabled-by-default TensorFlow features that we don't care about.
+BAZEL_BUILD_FLAGS+=(--define=no_aws_support=true)
+BAZEL_BUILD_FLAGS+=(--define=no_gcp_support=true)
+BAZEL_BUILD_FLAGS+=(--define=no_hdfs_support=true)
+BAZEL_BUILD_FLAGS+=(--define=no_kafka_support=true)
+BAZEL_BUILD_FLAGS+=(--define=no_ignite_support=true)
+BAZEL_BUILD_FLAGS+=(--define=grpc_no_ares=true)
+
+BAZEL_BUILD_FLAGS+=(--define=llvm_enable_zlib=false)
+BAZEL_BUILD_FLAGS+=(--verbose_failures)
+
+if [[ "${bb_full_target}" == *darwin* ]]; then
+    BAZEL_BUILD_FLAGS+=(--define=build_with_mkl=false --define=enable_mkl=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_x86_32_1=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_x86_64_1=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_x86_64_2=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_aarch64=false)
+    BAZEL_BUILD_FLAGS+=(--define=clang_macos_x86_64=true)
+    BAZEL_BUILD_FLAGS+=(--macos_minimum_os=10.14)
+    export MACOSX_DEPLOYMENT_TARGET=10.14
+    BAZEL_BUILD_FLAGS+=(--action_env=MACOSX_DEPLOYMENT_TARGET=10.14)
+    BAZEL_BUILD_FLAGS+=(--host_action_env=MACOSX_DEPLOYMENT_TARGET=10.14)
+    BAZEL_BUILD_FLAGS+=(--repo_env=MACOSX_DEPLOYMENT_TARGET=10.14)
+    BAZEL_BUILD_FLAGS+=(--test_env=MACOSX_DEPLOYMENT_TARGET=10.14)
+    BAZEL_BUILD_FLAGS+=(--noincompatible_remove_legacy_whole_archive)
+    BAZEL_BUILD_FLAGS+=(-s)
+    env
+fi
+
+if [[ "${bb_full_target}" == *freebsd* ]]; then
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_x86_32_1=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_x86_64_1=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_x86_64_2=false)
+    BAZEL_BUILD_FLAGS+=(--define=gcc_linux_aarch64=false)
+    BAZEL_BUILD_FLAGS+=(--define=freebsd=true)
+fi
+
+if [[ "${bb_full_target}" == *i686* ]]; then
+    BAZEL_BUILD_FLAGS+=(--define=build_with_mkl=false --define=enable_mkl=false)
+fi
+
+# julia --project=. -e "using Pkg; Pkg.instantiate(); Pkg.add(url=\"https://github.com/JuliaInterop/Clang.jl\", rev=\"vc/cxx_parse2\")"
+BAZEL_BUILD_FLAGS+=(--action_env=JULIA=julia)
+bazel ${BAZEL_FLAGS[@]} build ${BAZEL_BUILD_FLAGS[@]} ...
+rm bazel-bin/libReactantExtraLib*
+mkdir -p ${prefix}/lib
+cp bazel-bin/libReactantExtra* ${prefix}/lib
```

Comment:
Minor nitpick to make `cp` command more verbose and see what's happening
```suggestion
mkdir -p ${libdir}
cp -v bazel-bin/libReactantExtra* ${libdir}
```